.github/workflows/cd-frontend.yml

```diff
@@ -20,6 +20,7 @@ on:
         type: choice
         options:
           - dev
+          - staging
```

Comment:
Unfortunately adding `staging` here won't change much. It just makes staging available as an option, but doesn't actually start deploying staging. The line you want to change is down below...